jwt.go

```diff
@@ -0,0 +1,67 @@
+package clerk
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/go-jose/go-jose/v3/jwt"
+)
+
+type key string
+
+const activeSessionClaims = key("activeSessionClaims")
+
+// ContextWithSessionClaims returns a new context which includes the
+// active session claims.
+func ContextWithSessionClaims(ctx context.Context, value any) context.Context {
+	return context.WithValue(ctx, activeSessionClaims, value)
+}
+
+// SessionClaimsFromContext returns the active session claims from
+// the context.
+func SessionClaimsFromContext(ctx context.Context) (*SessionClaims, bool) {
+	claims, ok := ctx.Value(activeSessionClaims).(*SessionClaims)
+	return claims, ok
+}
+
+// SessionClaims represents Clerk specific JWT claims.
+type SessionClaims struct {
+	jwt.Claims
+	SessionID                     string          `json:"sid"`
+	AuthorizedParty               string          `json:"azp"`
+	ActiveOrganizationID          string          `json:"org_id"`
+	ActiveOrganizationSlug        string          `json:"org_slug"`
+	ActiveOrganizationRole        string          `json:"org_role"`
+	ActiveOrganizationPermissions []string        `json:"org_permissions"`
+	Actor                         json.RawMessage `json:"act,omitempty"`
+}
+
+// HasPermission checks if the session claims contain the provided
+// organization permission.
+// Use this helper to check if a user has the specific permission in
+// the active organization.
+func (s *SessionClaims) HasPermission(permission string) bool {
```

Comment:
❓ Aren't there some unit tests for these methods? Shouldn't we transfer them as well? Perhaps I've missed it though.

Reply:
:+1: Added new tests.